src/core/sagas/initializeSagas.js

```diff
@@ -0,0 +1,31 @@
+
+import { takeEvery, put } from 'redux-saga/effects';
+import  { get } from "lodash";
+
+import { domainName } from "../token";
```

Comment:
use window.config instead

Reply:
I did this in previous PR.

But I did following:
1) **domainName** is imported from "../token";
2) domainName defined by condition (window && window.config....)

Can I leave this variant?
Because in this case we can:
1) don't copy _(window && window.config....)_ in each place, where we use domainName;
2) if we change name of global variable **domainName** - we will have only one place, where it be used.

Reply:
Yes, that's all right